### ISSUE TITLE
`final Node chartBackground = chart.lookup(".chart-plot-background");  for what?

### ISSUE THREAD

**Issue by TututuXi** [kerner1000/javafx-chart-zooming#1]
Hi, I need the zoom function so I added your `ZoomManager` class in my application.  And I made some changes to it. But when I added my chart in my pane (chart in StackPane in VBox in GridPane in Scene) and zoom, the chart disappeared. So I want to kown your code how to work. However I have a question that I can not  understand. 
1. In line 233  `final Node chartBackground = chart.lookup(".chart-plot-background");` Do this for what?

**Comment by kerner1000** [comment]
Hi,
please see the [Examples](https://github.com/kerner1000/javafx-chart-zooming/blob/master/src/main/java/com/github/javafx/charts/zooming/BarChartSample.java).

```
// DO NOT ADD DATA TO CHART
// bc.getData().addAll(series1, series2, series3);
final StackPane pane = new StackPane();
pane.getChildren().add(bc);
final Scene scene = new Scene(pane, 500, 400);
new ZoomManager(pane, bc, series1, series2, series3);
```

Do not add the chart data to the chart, but to the zoom manager.

**Comment by kerner1000** [comment]
Fixed with cd931bd9ae002054ddc4ebf616d04677a103ac6c

**Comment by lhuangjs** [comment]
```
chart.setOnMouseReleased(new EventHandler<MouseEvent>() {
	@Override
	public void handle(final MouseEvent event) {
	final Bounds bb = chartBackground.sceneToLocal(rect.getBoundsInLocal());
	final double minx = bb.getMinX();
	final double maxx = bb.getMaxX();
	final double miny = bb.getMinY();
	final double maxy = bb.getMaxY();
	doZoom(true, chart.getXAxis().getValueForDisplay(minx), chart.getXAxis().getValueForDisplay(maxx));
	doZoom(false, chart.getYAxis().getValueForDisplay(miny), chart.getYAxis().getValueForDisplay(maxy));
	rect.setWidth(0);
	rect.setHeight(0);
	}
});
```
when I print `miny` and `maxy`, I get negative. Why?
